build.gradle

```diff
@@ -3,7 +3,9 @@ subprojects { project ->
   version = VERSION_NAME
 
   repositories {
-    mavenCentral()
+    maven {
+      url "https://plugins.gradle.org/m2/"
+    }
```

Comment:
@JakeWharton you were already using it here(https://github.com/JakeWharton/butterknife/pull/749/files#diff-c197962302397baf3a4cc36463dce5eaR34). This mirrors both `mavenCentral()` and `jcenter()`.